boot/environ/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ include_directories(BEFORE include/efi)
 include_directories(${REACTOS_SOURCE_DIR}/ntoskrnl/include)
 include_directories(${REACTOS_SOURCE_DIR}/sdk/lib/cmlib)
 include_directories(${REACTOS_SOURCE_DIR}/sdk/include/reactos/libs)
+include_directories(${REACTOS_SOURCE_DIR}/sdk/include/reactos/uefi)
```

Comment:
I suppose that this change would better belong to a separate commit in a separate PR. I don't see yet any moving around of files or sthg related to uefi out of boot/environ.